sql-plugin/src/main/scala/org/apache/spark/sql/execution/datasources/rapids/GpuPartitioningUtils.scala

```diff
@@ -40,6 +64,9 @@ object GpuPartitioningUtils {
    *
    * Mainly copied from PartitioningAwareFileIndex.inferPartitioning
    */
+    // Copied from Spark: org/apache/spark/sql/execution/datasources/PartitionUtils.scala
+    // for for https://github.com/NVIDIA/spark-rapids/issues/6026
+    // It can be removed when Spark3.3 is the least supported Spark version
```

Comment:
spacing off here, remove 2 spaces, really also should be combined with comment on line 65

Reply:
This comment shouldn't be here, it's my copy-paste mistake. Removed.